platform/persistence/platform-persistence-core-impl/src/main/java/org/codice/ddf/persistence/internal/PersistentStoreImpl.java

```diff
@@ -61,38 +65,43 @@ public PersistentStoreImpl(SolrClientFactoryImpl clientFactory) {
     }
 
     @Override
-    // Input Map is expected to have the suffixes on the key names
-    public void add(String type, Map<String, Object> properties) throws PersistenceException {
+    public void add(String type, Collection<Map<String, Object>> items) throws PersistenceException {
         LOGGER.debug("type = {}", type);
-        if (type == null || type.isEmpty()) {
+        if (StringUtils.isEmpty(type)) {
             throw new PersistenceException(
                     "The type of object(s) to retrieve must be non-null and not blank, e.g., notification, metacard, etc.");
         }
-        if (properties == null || properties.isEmpty() || properties.containsValue("guest")) {
+        if (CollectionUtils.isEmpty(items)) {
             return;
         }
 
-        LOGGER.debug("Adding entry of type {}", type);
-
         // Set Solr Core name to type and create solr client
         SolrClient solrClient = getSolrClient(type);
         if (solrClient == null) {
             throw new PersistenceException("Unable to create Solr client.");
         }
+        List<SolrInputDocument> inputDocuments = new ArrayList<>();
+        for (Map<String, Object> properties : items) {
+
+            if (MapUtils.isEmpty(properties)) {
+                continue;
+            }
 
-        Date now = new Date();
-        //DateFormat df = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ");
-        //String createdDate = df.format(now);
+            LOGGER.debug("Adding entry of type {}", type);
 
-        SolrInputDocument solrInputDocument = new SolrInputDocument();
-        solrInputDocument.addField("createddate_tdt", now);
+            Date now = new Date();
```

Comment:
❓ Should we inline this since the variable isn't used anywhere other than on L95?